hearts/tests/fake.py

```diff
@@ -11,8 +12,7 @@ def players(names='Lauren,Erin,Jeremy,Daniel'):
 
 def new_round(players_list=None, trick_plays=None, pass_to='left'):
     # trick_plays is in the form: [(leader_position, trick_string)]
-    if not players_list:
-        the_players = players()
+    the_players = players() if not players_list else players_list
```

Comment:
A good shorthand for this is `the_players = players_list or players()`. Cf. https://en.wikipedia.org/wiki/Short-circuit_evaluation